bn-apps/memberships-management/membership-service/src/test/kotlin/com/r3/businessnetworks/membership/customizations/DummyMembershipAutoAcceptor.kt

```diff
@@ -0,0 +1,12 @@
+package com.r3.businessnetworks.membership.customizations
```

Comment:
Looks like this is a separate change?

Reply:
For some reason git shows this change as a new file, but in fact I've just moved it between the packages.